src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -71,6 +71,25 @@ public UInt160[] GetBlockedAccounts(StoreView snapshot)
                 ?? Array.Empty<UInt160>();
         }
 
+        public bool IsAnyAccountBlocked(StoreView snapshot, params UInt160[] hashes)
+        {
+            if (hashes.Length == 0) return false;
+
+            var blockedList = snapshot.Storages.TryGet(CreateStorageKey(Prefix_BlockedAccounts))
+                ?.GetSerializableList<UInt160>().ToArray()
+                ?? Array.Empty<UInt160>();
+
+            if (blockedList.Length > 0)
+            {
+                foreach (var acc in hashes)
+                {
+                    if (Array.BinarySearch(blockedList, acc) != -1) return true;
```

Comment:
```suggestion
                    if (Array.BinarySearch(blockedList, acc) >= 0) return true;
```

The return value of `Array.BinarySearch` can be less then `-1`.